src/loaders/tvg/tvgTvgLoadParser.cpp

```diff
@@ -151,104 +155,100 @@ static LoaderResult _parseSceneProperty(tvgBlock block, Scene *scene)
     return result;
 }
 
-// Shape
-static LoaderResult _parseShapePath(const char *pointer, const char *end, Shape *shape)
+
+static LoaderResult _parseShapePath(const char *ptr, const char *end, Shape *shape)
 {
-    // ShapePath
+    //Shape Path
```

Comment:
necessary?

Reply:
trivial. rater than ShapePath.

Reply:
I meant, that I'd remove the comment :) but as you wish